Kickstarter-iOS/Locales/de.lproj/Localizable.strings

```diff
@@ -70,7 +71,7 @@
 "Closes_project" = "Schließt das Projekt.";
 "Collapses_subcategories" = "Reduziert die Anzeige der Unterkategorien.";
 "Collections" = "Sammlungen";
-"Comment_reply_digest" = "Comment reply digest";
+"Comment_reply_digest" = "Tägliche Zusammenfassung Kommentare";
```

Comment:
@dusi right now we don't have a way to specify which strings to download with the `make strings` script. When you run the script, it pulls down all changes to localized strings.

Reply:
I know 😞 

It just feels weird that our limited tooling allows for unrelated changes to make it to this PR. Would it be hard to remove using Xcode's DIFF comparison editor or any other GUI client for git? Unfortunately it's a manual process but it could be simplified if we only chose to commit strings we're interested in (given there is far less of those than our script has changed)

<img width="1277" alt="Screen Shot 2019-06-17 at 9 28 35 AM" src="https://user-images.githubusercontent.com/387596/59620742-774c2b80-90e2-11e9-802e-710b11e7a61a.png">

Reply:
I imagine the intention with this workflow was that all user interfaces should be flexible enough to support string updates as they come. If we're going to stop updating strings we need to come up with a new process for translators to request that we update specific strings in specific parts of the app as needed. My concern with "cherry-picking" strings (besides the fact that it's super manual) is that we'd be ignoring string changes that have intentionally been added by the international team. I think this is a much larger process topic given that the way it's worked in the past is we always update all strings so there isn't any communication between the translation team and us about what needs to be updated because they just update the string on their end and it gets updated on our end the next time someone runs `make strings`.

Reply:
I reverted all the other strings but I do think we should have a wider discussion about this before committing to changing our workflow.